pkg/notes/notes.go

```diff
@@ -1086,3 +1086,41 @@ func (rn *ReleaseNote) ContentHash() (string, error) {
 	}
 	return fmt.Sprintf("%x", h.Sum(nil)), nil
 }
+
+// Validate checks the data defined in a CVE map is complete and valid
+func (cve *CVEData) Validate() error {
+	if cve.CVSSRating == "" {
+		return errors.New("CVSS rating missing from CVE data")
+	}
+
+	// Check rating is a valid string
+	if cve.CVSSRating != "None" &&
+		cve.CVSSRating != "Low" &&
+		cve.CVSSRating != "Medium" &&
+		cve.CVSSRating != "High" &&
+		cve.CVSSRating != "Critical" {
+		return errors.New("Invalida CVSS rating")
+	}
+
+	if cve.CVSSVector == "" {
+		return errors.New("CVSS vector string missing from CVE data")
```

Comment:
Is there a library that can validate the form is the expected CVSS:3.1/AV:N/AC:H/PR:H/UI:R/S:U/C:H/I:H/A:H sort of form?

Reply:
I'm trying this one and seems to do the job just fine: https://github.com/spiegel-im-spiegel/go-cvss